internal/plugins/order/etcdraft/txpool/txpool.go

```diff
@@ -91,31 +102,29 @@ func (tp *TxPool) PoolSize() int {
 	return tp.pendingTxs.Len()
 }
 
-//Remove stored transactions
+//RemoveTxs remove txs from the cache
 func (tp *TxPool) RemoveTxs(hashes []types.Hash, isLeader bool) {
-	if isLeader {
-		tp.BatchDelete(hashes)
-	}
+	tp.Lock()
+	defer tp.Unlock()
```

Comment:
Why need to expand lock section？

Reply:
BatchDelete will spend more time to delete the transaction from the leveldb store, tp.lock just for locking the pendingtxs list.